src/pve_exporter/collector/cluster.py

```diff
@@ -203,6 +203,10 @@ def collect(self):  # pylint: disable=missing-docstring
                 'pve_storage_shared',
                 'Whether or not the storage is shared among cluster nodes',
                 labels=['id']),
+            'template': GaugeMetricFamily(
+                'pve_template',
+                'Whether or not the VM/CT is template',
+                labels=['id']),
```

Comment:
This adds a new _metric_. Judging from the feature request #199 I believe that a new _label_ should be added on the `pve_guest_info` metric.

Reply:
Maybe just try to add the `template` key into the `labels` array instead? Look for that a couple of lines further down.

Reply:
Hi,

I've update the template key as labels in pve_guest_info.
I'm sorry if there is any mistake in my code, because I'm not a developer.

Reply:
Perfect. Thanks, this is working well against my cluster.